test/framework/k8s_utils.go

```diff
@@ -219,6 +220,66 @@ func DeleteResource(ctx context.Context, k8sClient kubernetes.Interface, resourc
 	return err
 }
 
+// ScaleDeployment scales a deployment and waits until it is scaled
+func ScaleDeployment(setupContextTimeout time.Duration, client client.Client, desiredReplicas *int32, name, namespace string) (*int32, error) {
```

Comment:
This is not "setup-related", right?
```suggestion
func ScaleDeployment(timeout time.Duration, client client.Client, desiredReplicas *int32, name, namespace string) (*int32, error) {
```